flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -302,6 +303,20 @@ public class RocksDBNativeMetricOptions implements Serializable {
                     .withDescription(
                             "Monitor the duration of writer requiring to wait for compaction or flush to finish in RocksDB.");
 
+    public static final ConfigOption<Boolean> MONITOR_DB_GET =
+            ConfigOptions.key("state.backend.rocksdb.metrics.db_get")
```

Comment:
Please follow the metrics options style in this class, use `state.backend.rocksdb.metrics.db-get`.